### PR TITLE
Add exhale recipe

### PR DESCRIPTION
**Pull request by dillon-cullinan**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/exhale/exhale/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{% set name = "exhale" %}
```
Comment (mariusvniekerk) [inline recipes/exhale/exhale/meta.yaml:1]: incorrect filename, should be `recipes/exchale/meta.yaml`

**Comment by rigzba21** [comment]
confirming I am willing to be listed  as a maintainer

File: recipes/exhale/exhale/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    - python
+    - six
+    - sphinx >=1.6.1
+  run:
+    - breathe
+    - bs4
+    - lxml
+    - python
+    - six
+    - sphinx >=1.6.1
```
Comment (mariusvniekerk) [inline recipes/exhale/exhale/meta.yaml]: Grayskull gets these deps as

```suggestion
requirements:
  host:
    - pip
    - python >=3.6
  run:
    - beautifulsoup4
    - breathe >=4.32.0
    - docutils >=0.12
    - lxml
    - python >=3.6
    - six
    - sphinx >=3.0,<5
```

Comment (dillon-cullinan) [reply recipes/exhale/exhale/meta.yaml]: I also used grayskull to generate this using the following command locally: 
```
grayskull pypi exhale=0.2.3
```

**Pull request merged**